libmscore/noteentry.cpp

```diff
@@ -122,13 +122,13 @@ NoteVal Score::noteValForPosition(Position pos, bool &error)
 Note* Score::addPitch(NoteVal& nval, bool addFlag)
       {
       if (addFlag) {
-            Chord* c = toChord(_is.lastSegment()->element(_is.track()));
```

Comment:
there's an implicit Q_ASSERT in the toChord macro to ensure not null.  But since the if statement below this line already deals with testing for null, I'm delaying the toChord till after.